
 The if ENABLE flags don't match up. If we don't enable LLINT we
 get errors about callToJavaScript not being defined.

 And I don't want to get deeper into this. (Bigger fish and stuff...)
 (And why NOT enable it?)

--- a/Source/WTF/wtf/Platform.h	2013-11-30 20:51:00.000000000 +0100
+++ b/Source/WTF/wtf/Platform.h	2013-11-30 20:51:00.000000000 +0100
@@ -743,9 +743,9 @@
 /* On some of the platforms where we have a JIT, we want to also have the 
    low-level interpreter. */
 #if !defined(ENABLE_LLINT) \
     && ENABLE(JIT) \
-    && (OS(DARWIN) || OS(LINUX)) \
+    && (OS(DARWIN) || OS(LINUX) || OS(FREEBSD)) \
     && (PLATFORM(MAC) || PLATFORM(IOS) || PLATFORM(GTK)) \
     && (CPU(X86) || CPU(X86_64) || CPU(ARM_THUMB2) || CPU(ARM_TRADITIONAL) || CPU(ARM64) || CPU(MIPS) || CPU(SH4))
 #define ENABLE_LLINT 1
 #endif
